### PR TITLE
Added article related to create Hangout bot

### PR DESCRIPTION
**Pull request by pareek-naren**
Added article to create a Google Hangout Bot with Express and Node.js

### CODE REVIEW

**Comment by pareek-naren** [comment]
@rajikaimal Please review the PR

**Comment by pareek-naren** [comment]
@rajikaimal Please review the PR

File: README.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ Curated list of express.js resources
 - [How to uncouples emit events from connection event into socket.io and express](https://dev.to/wakeupmh/how-to-decouples-emit-events-from-connection-event-into-socket-io-8dk) - Uncoupling socket emit events with express
 - [LinkedIn Login using Node JS, Express and passport](https://www.loginradius.com/engineering/blog/linkedin-login-using-node-passport/)
```
Comment (rajikaimal) [inline README.md]: Hi @pareek-narendra can you please add descriptions of these articles with a single sentence in-front of the link. Thanks!

**Review by rajikaimal — COMMENTED** [review]
Hi @pareek-narendra can you please add descriptions of these articles with a single sentence in-front of the link. Thanks!

**Comment by pareek-naren** [comment]
@rajikaimal Added the suggested changes. Please review.

**Pull request merged**